src/test/java/org/myrobotlab/framework/ServiceLifeCycleTest.java

```diff
@@ -41,9 +41,20 @@ public static void setUpAfterClass() throws Exception {
     Runtime.release("thower02");
   }
 
-
   @Test
   public void serviceLifeCycleTest() throws Exception {
+        
+    // push the meta data into the planStore - this will persist 
+    // overrides if the meta data definition has them
+    ServiceData.setMetaData("i01", "InMoov");
+    
+    // pull it back out
+    MetaData data = ServiceData.getMetaData("i01", "InMoov");
+    log.warn("static meta data {}", data);
+
+    Plan plan = ServiceData.getPlan("i01", "InMoov");
+    log.warn("static plan {}", plan);
```

Comment:
Lets add some sort of an assert.. at a minimum that the plan isn't null.

Reply:
added a couple statements and noticed - logging level was too high